webapp/src/components/schedule/WeekCalendar.js

```diff
@@ -74,6 +77,38 @@ class WeekCalendar extends PureComponent {
         calendar.download("Calendar");
     }
 
+    addToGCalendar(subsections) {
+
+        let events = [];
+        const googleTimeFormat = "YYYY-MM-DDTHH:mm:ssZ";
+        let numWeeksAhead = 5;
+
+        let recurringEventRule = "RRULE:FREQ=WEEKLY;COUNT=" + numWeeksAhead;
```

Comment:
const and remove console log - removing log doesn't matter too much cause I override them but ideally if you need them should have some clarifying context information